### ISSUE TITLE
'High Contrast Wgite' theme Avatar (Zoom Icon) color

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1031]
**Describe the bug**
Zoom Icon variant in the Avatar component has the same text and background-color: black

> What version of the Fundamental Library Styles are you using?
v0.10.0-rc.1

**To Reproduce**
1. Go to https://deploy-preview-1025--fundamental-styles.netlify.app/components/avatar.html
2. Check the Zoom Icon component

**Expected behavior**
Zoom Icon component in the Avatar component should have a `color: #fff`

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81855491-45bb2a80-9514-11ea-97d2-4cd86d08fe1d.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
Fixed as of version 0.13.0 - Contrast ratio now 5:1
<img width="747" alt="Screen Shot 2020-10-07 at 5 23 25 PM" src="https://user-images.githubusercontent.com/21978807/95529782-b580f480-0990-11eb-851e-b3045b5af98c.png">
